open_diloco/train_fsdp.py

```diff
@@ -69,26 +70,18 @@ def log(message):
     logger.info(f"[rank {os.environ['LOCAL_RANK']}] {message}")
 
 
-def get_ckpt_folder(checkpoint_path, training_date, project, run_id):
-    return os.path.join(checkpoint_path, training_date, project, run_id)
-
-
-def check_checkpoint_path_access(checkpoint_path: str, training_date, project, run_id, rank):
-    dummy_file_path = os.path.join(
-        get_ckpt_folder(
-            checkpoint_path=checkpoint_path,
-            training_date=training_date,
-            project=project,
-            run_id=run_id,
-        ),
-        f"dummy_file_{rank}.txt",
-    )
+def check_checkpoint_path_access(checkpoint_path: str, rank: int):
+    dummy_file_path = os.path.join(checkpoint_path, f"dummy_file_{rank}.txt")
     with fsspec.open(dummy_file_path, "w") as f:
         f.write("This is a dummy file for testing access.")
     gfs = GenericFileSystem()
     gfs.rm(dummy_file_path)
 
 
+def get_diloco_rank_path(world_rank_diloco: int):
+    return os.path.join(f"diloco_rank_{world_rank_diloco}")
```

Comment:
Seems like the os.path.join here is unnecessary?

Reply:
its more a folder / directory name than a path. maybe the function name could be different to reflect that.

Reply:
yeah good point, i was doing an actual join before but I removed it. Will update